lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -844,7 +844,8 @@ void ConfigBuilder::buildVsRegConfig(ShaderStage shaderStage, T *pConfig) {
   bool enableXfb = resUsage->inOutUsage.enableXfb;
   if (shaderStage == ShaderStageCopyShader) {
     // NOTE: For copy shader, we use fixed number of user data registers.
-    SET_REG_FIELD(&pConfig->vsRegs, SPI_SHADER_PGM_RSRC2_VS, USER_SGPR, lgc::CopyShaderUserSgprCount);
+    auto copyShaderUserSprgCount = lgc::CopyShaderUserSgprCount;
```

Comment:
Better to make a separate pull request for this change. It is different from others. Also, better to add explanations in the commit message to describe why you do this since it is very strange for external audience.